### PR TITLE
Fix/6768 adding missing screenshots 2.1 release

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Adding a launch argument to add missing screenshot for test result cards

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6768

## Screenshots
Simulator Screen Shot - Clone 1 of iPhone 11 - 2021-04-28 at 18 32 27

### CODE REVIEW

File: src/xcode/ENA/ENAUITests/Home/ENAUITestsHome.swift [hunk 1]
```diff
@@ -366,6 +366,18 @@ class ENAUITests_01_Home: XCTestCase {
 		snapshot("details_screen_risk_level_\(riskLevel)_risk_one_day_installation_\(installationDays)days_\(String(format: "%04d", (screenshotCounter.inc() )))")
 	}
 	
+	func test_screenshot_homescreen_pcr_rat_negative() throws {
+		app.setPreferredContentSizeCategory(accessibility: .accessibility, size: .XS)
+		app.launchArguments.append(contentsOf: ["-showTestResultCards", TestResult.negative.stringValue])
```
Comment (marcussc) [inline src/xcode/ENA/ENAUITests/Home/ENAUITestsHome.swift:371]: Shouldn't this be something like: 
```suggestion
		app.launchArguments.append(contentsOf: ["-showTestResultCards", "YES"])
```
?

File: src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift [hunk 2]
```diff
@@ -127,6 +127,12 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 	override func viewDidAppear(_ animated: Bool) {
 		super.viewDidAppear(animated)
 
+		#if DEBUG
+		if isUITesting && UserDefaults.standard.string(forKey: "showTestResultCards") == "YES" {
+			tableView.scrollToRow(at: IndexPath(row: 1, section: 1), at: .top, animated: false)
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift:132]: Is this needed? Why is the UI-Test process not "scrolling" to this position on its own?

Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift:132]: Yes, it is needed. For some reason, wait for existence is not working, and the required screenshot like both cards on the same view can not be achieved by swipeup/swipedown, hence we need to come up with this.

Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift:132]: Unfortunately, we don’t have the scroll to or some similar function while UI testing.

**Review by marcussc — APPROVED** [review]

**Pull request merged**